pkg/storage/mvcc_history_test.go

```diff
@@ -168,14 +212,9 @@ func TestMVCCHistories(t *testing.T) {
 				// "error" means expect an error to occur. The specific error type/
 				// message to expect is spelled out in the expected output.
 				//
```

Comment:
nit: Explanation of new option stats is missing here and above in DSL section.

Reply:
Added, thanks.